### ISSUE TITLE
OneNote import "Cannot read properties of undefined (reading 'split')"

### ISSUE THREAD

**Issue by jenschristiansson** [obsidianmd/obsidian-importer#319]
Importing from OneNote it will not import many notes and I get the error:
Cannot read properties of undefined (reading 'split')

**Comment by balmyfalse** [comment]
> Importing from OneNote it will not import many notes and I get the error: Cannot read properties of undefined (reading 'split')

Same for me.

**Comment by keichhorn** [comment]
I also have the same Problem :-(

**Comment by tgrosinger** [comment]
Can you please provide any lots shown in the console when this error is encountered? Press ctrl+shift+i to open the console. Thank you!

**Comment by keichhorn** [comment]
@tgrosinger Unfortunately I haven't noted which Onenote Notebook I have tried to import at that time. (And I didn't know that the console can be opened with Ctrl + Shift  + i)

I have just tested importing different Onenote Notebooks (I can't import everything because of Microsoft Rate Limiting, and because my Notebooks are huge).

My findings with the new Importer Version 1.6.15: 
The Onenote Import works like a charm. I haven't encountered the Error " Cannot read properties of undefined (reading 'split') " with the new Version. And all Text/Images are fully imported as far as I can see.

Therefore, I think this Issue is solved.
